packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -1792,14 +1795,14 @@ export interface GarbageCollectionOptions {
    *
    * @default 'full'
    */
-  readonly action: 'print' | 'tag' | 'delete-tagged' | 'full';
+  readonly action: ActionType;
```

Comment:
```suggestion
  readonly action: GarbageCollectionAction;
```

Reply:
im reverting these changes. tying them as `any` as they were before. can revisit in separate PR if we want to handle all cases where we turn a string into an enum in a better way